Cooper/ClientApp/src.v2/app/layouts/top-panel/top-panel.layout.ts

```diff
@@ -3,9 +3,10 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'coop-top-panel-layout',
   templateUrl: './top-panel.layout.html',
-  styleUrls: ['./top-panel.layout.css']
+  styleUrls: ['./top-panel.layout.scss']
 })
 export class TopPanelLayoutComponent implements OnInit {
+  elements = [{label: 'Home', link: '#'}, {label: 'Vacans', link: '#'}, {label: 'Library', link: '#'}, {label: 'Teams', link: '#'} ]
```

Comment:
please, each array's element on separate line
navigationItems = [
  {label: 'Home', link: '#'},
  {label: 'Vacancies', link: '#'},
  ...
]
also, please, add links for existing pages and add items for chats, forums, my profile

Reply:
instead of Library it should be Games it seems

Reply:
also without teams
and games should be before vacancies
might, order can be next:
home,
games,
chats,
my profile,
forum,
vacancies

Reply:
in future my profile go to other place